### PR TITLE
Issue180/deprecate uv data to dict

### PR DESCRIPTION
**Pull request by zakiali**
This PR changes the instances of the UVData_2_dict function in firstcal to use the io.load_vis function.

Also deprecates `abscal.UVData2AbscalDict` for `io.load_vis`

### CODE REVIEW

**Comment by nkern** [comment]
addresses #179 and #180

File: hera_cal/abscal_funcs.py [hunk 1]
```diff
@@ -1563,7 +1462,7 @@ def avg_file_across_red_bls(data_fname, outdir=None, output_fname=None,
         uvd.read_miriad(data_fname)
 
     # get data
-    data, flags = UVData2AbsCalDict(uvd)
+    data, flags = io.load_vis(uvd, pop_autos=True, return_meta=False, pick_data_ants=True)
```
Comment (jsdillon) [inline hera_cal/abscal_funcs.py]: You don't need return_meta=False.

Comment (nkern) [reply hera_cal/abscal_funcs.py]: yeah, just in case defaults change doesn't hurt to have

**Comment by nkern** [comment]
ah, you merged it by accident... let me go back and undo that...

**Comment by nkern** [comment]
ok should be good to go now

**Comment by nkern** [comment]
*once it passes travis

**Pull request merged**